packages/peerplays/lib/peerplays.js

```diff
@@ -82,6 +86,28 @@ export default class PPY extends Plugin {
     );
   }
 
+  /**
+   * Generate keys role=("owner"|"active"|"memo") from (password + accountName + role)
+   *
+   * @param accountName String
+   * @param password String
+   * @param roles Array
+   * @returns Object of roles
```

Comment:
this is incorrect, it's returning a Keypair Object

Reply:
also defining the type of the param wrong, here is a correct definition:
![image](https://user-images.githubusercontent.com/36667230/74867381-f60cfe80-532a-11ea-8e1a-73c31fb42d8a.png)